x/liquidity/types/utils.go

```diff
@@ -72,7 +72,18 @@ func ValidateReserveCoinLimit(maxReserveCoinAmount sdk.Int, depositCoins sdk.Coi
 }
 
 func GetOfferCoinFee(offerCoin sdk.Coin, swapFeeRate sdk.Dec) sdk.Coin {
-	// apply half-ratio swap fee rate
+	if swapFeeRate.IsZero() {
```

Comment:
Doesn't this return 0 without the if statement?

Reply:
If we need this if statement, then we might as well cover this case in `utils_test.go`
```go
{
  name:               "case8",
  offerCoin:          sdk.NewCoin(testDenom, sdk.NewInt(10000)),
  swapFeeRate:        sdk.ZeroDec(),
  expectOfferCoinFee: sdk.NewCoin(testDenom, sdk.NewInt(0)),
},
```